docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+This SDK supports a minimum Android API of 23 or higher.
+Android apps can be written in Kotlin or Java 8 or higher.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
+
+### Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
+Follow the steps in [Enable the SDK](https://developer.paypal.com/sdk/in-app/android/#link-enablethesdk) to set up your account and application for PayPal payments.
```

Comment:
Just wanted to note that there are still a lot of steps required to set up this integration. Following all of the steps in these two instruction steps was pretty time consuming, and it was easy to miss small steps and have the integration not work without a clear error message.

Reply:
Also I assume we'd update these links to dev docs pages specific to this SDK, but for now to not duplicate instructions I am just linking to the orders API and NXO setup instructions.

Reply:
I agree 100% with this 🙁  Unfortunately our hands are tied when it comes to Identity requirements. With the current way buyer authentication is set up, merchants will have to go through these setup steps for the PayPal integration.